src/app/modules/core/auth/models/user.ts

```diff
@@ -0,0 +1,101 @@
+import { Role } from 'src/app/shared/models/role.enum'
+
+export interface IName {
+  first: string
+  middle?: string
+  last: string
+}
+
+export enum PhoneType {
+  None = 'none',
+  Mobile = 'mobile',
+  Home = 'home',
+  Work = 'work',
+}
+
+export interface IPhone {
+  type: PhoneType
+  digits: string
+  id: number
+}
+
+export interface IUser {
```

Comment:
ther is another IUser interface in shared/models/users.ts

Reply:
I know, we'll do a refactor to align the models but good catch @lareas123 :)